vdr/doc/resolvers.go

```diff
@@ -64,7 +101,13 @@ func (d Resolver) ResolveControllers(doc did.Document, metadata *types.ResolveMe
 
 	// resolve all unresolved docs
 	for _, ref := range refsToResolve {
-		node, _, err := d.Store.Resolve(ref, metadata)
+		node, _, err := d.resolve(ref, metadata, depth)
+		if errors.Is(err, types.ErrDeactivated) {
+			continue
+		}
+		if errors.Is(err, ErrNestedDocumentsTooDeep) {
+			return nil, err
+		}
```

Comment:
Maybe it would be nice if we can return something like `ErrNoActiveController` or something similar when all of the controllers are deactivated.

Reply:
It could help in debugging since an `ErrNoActiveController` would generate some other logging.